facebook/__init__.py

```diff
@@ -143,7 +143,7 @@ def get_all_connections(self, id, connection_name, **args):
             args = parse_qs(urlparse(next).query)
             del args['access_token']
 
-    def put_object(self, parent_object, connection_name, **data):
+    def put_object(self, parent_object, connection_name, file=None, **data):
```

Comment:
Adding a new argument here means that the method API documentation also needs to be updated.